src/lib/components/Map.react.js

```diff
@@ -364,7 +367,18 @@ Map.propTypes = {
     /**
      * Dash callback property. Receives [lat, lng, accuracy] when user location is found.
      */
-    location_lat_lon_acc: PropTypes.arrayOf(PropTypes.number)
+    location_lat_lon_acc: PropTypes.arrayOf(PropTypes.number),
+
+
+    /**
+     * Dash callback property for viewport changed (zoom)
+     */
+    changed_zoom:  PropTypes.number,
```

Comment:
Is there a specific reason why you are introducing new properties rather than just setting the value of the `zoom ` property?

Reply:
I trief reusing the ```zoom``` and ```center``` properties. This would be of course much better!
But for some reason it didn't work.

Reply:
Thanks for the rapid comment! I'll give it a try :)

Reply:
Later (or maybe you want to change) I could try again with changing to correct property names...maybe I forgot to rebuild (a common error with these dash-components :)

Reply:
see my commit below